### PR TITLE
Build from source requires Ubuntu 18.04

### PR DESCRIPTION
**Pull request by chrishobcroft**
**What does this pull request do? Explain your changes. (required)**

This change clarifies that the `install_ffmpeg.sh` step of the `install.md` document requires `Ubuntu 18.04` to complete successfully.

Previous testing was done on `Ubuntu 20.04 LTS`, on which `install_ffmpeg.sh` was shown to consistently fail, as documented in https://github.com/livepeer/go-livepeer/issues/1500

**Specific updates (required)**

- Added that `18.04` version of Ubuntu is required

**How did you test each of these updates (required)**

Tested running the commands as shown below in the following environments, with these results:

|                       	| Ubuntu 18.04 LTS 	| Ubuntu 20.04 LTS 	|
|-----------------------	|:----------------:	|:----------------:	|
| Amazon AWS     	|      success     	|      not tested      	|
| Digital Ocean VPS     	|      success     	|      failed      	|
| Exoscale VPS          	|      success     	|      failed      	|
| Lenovo Thinkpad T490s 	|    not tested    	|      failed      	|

Commands run as `ubuntu` from clean install:
```
sudo apt-get update
sudo apt-get -y install build-essential pkg-config autoconf gnutls-dev git curl
mkdir livepeer && cd livepeer
git clone https://github.com/livepeer/go-livepeer.git
cd go-livepeer
./install_ffmpeg.sh
wget https://dl.google.com/go/go1.14.4.linux-amd64.tar.gz
sudo tar -C /usr/local -xzf go1.14.4.linux-amd64.tar.gz
export PATH=$PATH:/usr/local/go/bin
PKG_CONFIG_PATH=~/compiled/lib/pkgconfig make
```

**Does this pull request close any open issues?**

None.

**Checklist:**

- [x] README and other documentation updated
- [ ] Node runs in OSX and devenv
- [ ] All tests in `./test.sh` pass

### CODE REVIEW

File: doc/install.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ You can also build your own executables from source code.
 
 &ensp; 2\. Make sure you have the necessary libraries installed:
 
- * Linux (Ubuntu): `apt-get update && apt-get -y install build-essential pkg-config autoconf gnutls-dev git curl`
+ * Linux (Ubuntu 18.04): `apt-get update && apt-get -y install build-essential pkg-config autoconf gnutls-dev git curl`
```
Comment (yondonfu) [inline doc/install.md]: 16.04 should also work (this is the version used in this repo's [CI](https://github.com/livepeer/go-livepeer/blob/d0ea014d68488c1bc26c4e5882949a0d1dc1c86c/docker/Dockerfile.build-linux#L1)).

Could we include 16.04 here as well?

**Review by yondonfu — CHANGES_REQUESTED** [review]
Thanks for the thorough testing! Left a small comment about another Ubuntu version that is known to work.

File: doc/install.md (hunk 1 above)
Comment (chrishobcroft) [reply doc/install.md]: Thanks @yondonfu - added 16.04

**Review by yondonfu — APPROVED** [review]
LGTM!

**Pull request merged**